src/server/scripts/Pandaria/GateSettingSun/gate_setting_sun.hpp

```diff
@@ -0,0 +1,134 @@
+////////////////////////////////////////////////////////////////////////////////
+///
+///  MILLENIUM-STUDIO
```

Comment:
Millenium-Studios? I seriously doubt Firestorm released any of their scripts. with 2015, this has to be some sort of leak that can not be used. What exactly are you thinking?